src/classes/COFF_Affiliation_TDTM.cls

```diff
@@ -71,19 +71,28 @@ public class COFF_Affiliation_TDTM extends TDTM_Runnable {
             Map<Id, Course_Offering__c> mapNewFacultyIdToNewCourseOffering = new Map<Id, Course_Offering__c>();
 
             if(newlist != null) {
+                
                 if (triggerAction == TDTM_Runnable.Action.AfterInsert || triggerAction == TDTM_Runnable.Action.AfterUpdate) {
                     //Query fields not available in the trigger
-                    newList = [select Course__c, Faculty__c, Start_Date__c, End_Date__c, Course__r.Account__c, Term__r.Start_Date__c,
-                                Term__r.End_Date__c from Course_Offering__c where ID in :newlist];
+                    newList = [SELECT Course__c
+                                      , Faculty__c
```

Comment:
@lifewithryan Commas at the beginning of a line? 🙀

Reply:
haha, yea, it's hold over from what I've done for years. Actually learned it from Kevin Swiggum :)

We don't *have* to do it that way, but I definitely don't want the wrapping. If it's going to wrap I'd rather line break it and line up the keywords at a minimum.

Reply:
I agree about the wrapping! I'm not sure I'll adopt this way, but it's fine. I've just never seen it like this before.

Reply:
No worries. As long as we avoid wrapping/scrolling right and we use some formatting to keep it legible, I'm good. We have three very usable options in the style guide.